css/product-payment.css

```diff
@@ -0,0 +1,555 @@
+  font-weight: var(--font-weight);
+  line-height: 20.11px;
+  color: var(--color-dark-gray);
+}
+
+.product-page__ofer{
+  display: flex;
+  align-items: center;
+  gap: var(--ten-pixel);
+  font-size: var(--font-size-smalls);
+  font-weight: var(--font-weight);
+  line-height: var(--eighteen-pixel);
+  letter-spacing: -0.01em;
+  color: var(--color-gray);
+  border: 1px solid var(--color-soft-gray);
+  border-radius: var(--eight-pixel);
+  padding: 14px 15px 14px 10px;
+  text-decoration: none;
+}
+
+.product-page__add{
+  display: flex;
+  margin-top: 20px;
+  gap: 28px;
+  background-color: var(--color-whiteX);
+  padding: 19px 0px;
+  border-radius: 0px 0px 16px 16px;
+  position: relative;
+}  
+
+.product-page__quantity-selector{
+  display: flex;
+  align-items: center;
+  border: 1px solid var(--color-solid);
+  width: 115px;
+  border-radius: var(--eight-pixel);
+  justify-content: center;
+  gap: 5px;
+  position: absolute;
+  margin-left: 20px;
+  background-color: var(--color-white);
+}
+
+.product-page__quantity__link {
+  font-size: 0;
+  inset: 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.product-page__decrease{
+  border: none;
+  background: none;
+  cursor: pointer;
+  width: 22px;
+  border: 1px solid var(--color-solid);
+  margin-left: 5px;
+}
+
+.product-page__quantity {
+  font-size: var(--twenty-pixel);
+  font-weight: var(--font-weight);
+  line-height: 25px;
+  text-align: center;
+  width: 95px;
+  padding: 7px 10px 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 5px;
+  border-left: 1.5px solid var(--color-solid);
+  border-right: 1.5px solid var(--color-solid);
+  color: var(--color-solid);
+  background-color: var(--color-whiteX);
+}
+
+.product-page__increase{
+  border: none;
+  background: none;
+  cursor: pointer;
+  width: 25px;
+  border: 1.5px solid var(--color-solid);
+  margin-right: 6px;
+}
+
+.product-page__increase--1{
+  height: var(--eighteen-pixel);  
+  width: var(--zero);
+  position: absolute;
+  margin-left: 93px;
+}
+
+.product-page__add-purchases{
+  display: flex;
+  gap: var(--eight-pixel);
+  background-color: var(--color-pink);
+  border: none;
+  border-radius: var(--eight-pixel);
+  cursor: pointer;
+  box-shadow: 0px 4px 12px 0px #D14D411A;
+  padding: 10px 20px;
+  position: relative;
+  margin-top: -5px;
+  margin-right: 20px;
+}
+
+.product-page__add-purchases:hover{
+  background-color: #9e255b;
+}
+
+.product-page__add-cart{
+  font-size: var(--font-size-smalls);
+  font-weight: var(--font-weight-small);
+  line-height: var(--eighteen-pixel);
+  text-align: center;
+  color: var(--color-white);
+  letter-spacing: 0.06em;
+}
+
+.product-page__payment-methods {
+  display: flex;
+  height: 9px;
+  gap: 5px;
+  margin-top: 10px;
+}
+
+.product-page__add-target{
+  margin-left: 150px;
+}
+
+/* Desktop style */
+@media (min-width: 768px) {
+  .product-page__top {
+    padding-bottom: 24px;
+    margin: 0 5px; 
```

Comment:
En el diseño se ve alineada eta barra con el area de la imagen, no se ve separacion extra del borde

<img width="292" alt="image" src="https://github.com/user-attachments/assets/f7f3c92a-a116-44a8-a304-6f2b53785a20">

```suggestion
    margin: 0; 
```